index.html

```diff
@@ -284,7 +296,7 @@ <h4>SPARQL family of specifications</h4>
               <p class="draft-status"><b>Draft state:</b> Adopted from the <a href="https://www.w3.org/2009/sparql/wiki/Main_Page">SPARQL (1.1) WG</a></p>
               <p><b>Adopted Draft:</b> <a href="https://www.w3.org/TR/2013/REC-sparql11-query-20130321/">SPARQL 1.1 Query Language</a>, eds. Steve Harris, Andy Seaborne, W3C Recommendation.</p>
               <p class="draft-status"><b>Reference:</b> <a href="https://www.w3.org/2021/12/rdf-star.html#sparql-star">Section 4 of RDF-star and SPARQL-star</a>, eds. Olaf Hartig, Pierre-Antoine Champin, Gregg Kellogg, Andy Seaborne, W3C Community Group Final Report, 2021. </p>
-              <p class="milestone"><b>Expected completion:</b> 29 February 2024</p>
+              <p class="milestone"><b>Expected completion:</b> Q1 2025</p>
```

Comment:
Here and sparql-results-json, it's Q1 2025, but other SPARQl specs are Q2 and the timeline says "Q2 2025: Recommendation of SPARQL 1.2 deliverables".

```suggestion
              <p class="milestone"><b>Expected completion:</b> Q2 2025</p>
```